### ISSUE TITLE
5-02-01 Methods (atmosphere) - duplicated entries?

### ISSUE THREAD

**Issue by fierz** [wmo-im/wmds#167]
**Summary and Purpose**  
In code table 5-02-01 (Observing method - atmosphere) element 201 (thermocouple, thermophile) and element 202 (thermometer/thermoelement) are duplicates.

**Proposal**  
Deprecate element 202

**Reason**  
Keeping both elements is unnecessary and potentially confusing.

**Branch**  
[https://github.com/wmo-im/wmds/tree/issue-167](https://github.com/wmo-im/wmds/tree/issue-167)

**Decision**
Element 202 is superseded by 201. 

---

_\-original issue comment_  
While I wanted to choose a method for measuring air temperature, I came across these two entries:  
201, Thermocouple, thermophile  
202, Thermometer-Thermoelement

I could not figure out what the difference between both is, or , put it another way, I wonder whether notation 202 is not just a German translation for 201 (thermocouple => Thermoelement)?

**Comment by IgorZahumensky** [comment]
@joergklausen  @KarlBureau @fierz 
any ideas how to deal with?

**Comment by joergklausen** [comment]
I think @fierz is right, these are duplicates. Element 202 should be deprecated and removed as a duplicate. Can it still be done for this FT, I don't know?

**Comment by IgorZahumensky** [comment]
@amilan17 @efucile : still manageable for this FT, please?

**Comment by IgorZahumensky** [comment]
@amilan17 : please have a look at this issue if still manageable for this fast track, please.

**Comment by amilan17** [comment]
This is the branch: https://github.com/wmo-im/wmds/tree/issue-167/tables_en

**Comment by amilan17** [comment]
@fierz @joergklausen 
Can you please review the branch? I removed 202 from the table and listed it in the superseded.txt file.

**Comment by fierz** [comment]
@amilan17 fine with me. Thanks.

**Comment by joergklausen** [comment]
@amilan17 I confirm the entry 202 is missing from https://github.com/wmo-im/wmds/blob/issue-167/tables_en/5-02-01.csv. All fine.

**Comment by joergklausen** [comment]
@amilan17 This should have been dealt with and completed already. Can you please confirm that it was approved in FT-2020-2 and close the issue accordingly?

**Comment by amilan17** [comment]
This did not make it into the FT-2020-2. I added FT-2021-1 milestone.

**Comment by fstuerzl** [comment]
**Summary and Purpose**
In code table 5-02-01 (Observing method - atmosphere) element 201 (thermocouple, thermophile) and element 202 (thermometer/thermoelement) are duplicates.

**Proposal**
Deprecate element 202

**Reason**
Keeping both elements is unnecessary and potentially confusing.

**Comment by amilan17** [comment]
Dependent on #199

**Comment by joergklausen** [comment]
@RMaerz Please confirm consistency of branch and follow through.

**Comment by amilan17** [comment]
merged with FT-2021-1-validation branch
